src/SupervisionSuccess/model/calcOutcomesProportions.js

```diff
@@ -0,0 +1,59 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+/**
+ * Function that calculates outcome proportions from the given inputs.
+ * @param {number} changeInRevocations - integer value of the percentage change in revocations
+ * @param {number} changeInNewAdmissions - integer value of the percentage change in new admissions
+ * @const {number} prisonPopulationDiff - total difference of people in prison
+ * @returns {number}
+ */
+function calcOutcomesProportions(
```

Comment:
If I understand correctly, this is calculating the proportion of the change in the size of the population and then using that for both the population and cost numbers in the outcomes, right? But is it the case that the proportions for population and for cost will always be the same? Why don't we need to calculate proportions for cost independently?

Reply:
So, in case of population outcome we use population diff and in case of costs what values should we use for calculations?

Reply:
Mm, I may have been mistaken. Is the point that your updated model calculations only produce a single savings output? It doesn't produce savings from revocation change and also savings from new admission change?

@cawarren want to confirm that that sounds right to you. If so, then should the proportions for the cost outcome change be based on the population diff, and thus always be the same as the proportions for the population outcome change?

Reply:
Yes, total policy savings calculates right away from total population, which is the sum of revocations and new admissions. This is the case according to the model: https://docs.google.com/spreadsheets/d/1qa4Ts6Aw8MEpAN9vWLBeiUogRg_pmIKMnMxWKCcPOqA/edit#gid=1841382624

Reply:
My understanding is also that the cost calculation is solely determinant on population increase or decrease.

Reply:
Chatted with @cawarren offline and he agreed that the UI is not ideal as currently designed since the percentages are always the same under both top-line numbers. We're going to file a separate ticket for that. For now, we should keep calculating the proportions for cost the same as you are now, @nasaownsky, using the same numbers as for population, sticking with the current design. With that settled, you can wrap up the other comment thread above!

Andrew's comments for posterity:
>You're right that it feels a little wonky. I'm going to ping Juan to see if we can do a slightly different visual treatment, but nothing very radical (just consolidating to one set of %'s, below both of the numbers). Thanks for flagging it.